### ISSUE TITLE
include googld ads in custom skin loop?

### ISSUE THREAD

**Issue by karrolhk** [dudaster/ele-custom-skin#266]
Hi, can I use the custom skin to create a loop that display google ads every 5 posts?

Like this in the screenshot

![adsense loop](https://user-images.githubusercontent.com/64810663/86815602-d4e64980-c050-11ea-8ca7-e2e8470c36ff.jpg)

**Comment by jaykaslo** [comment]
Good question. I'd love to be able to do this too!

**Comment by dudaster** [comment]
You can use custom grid but without masonry

**Comment by jaykaslo** [comment]
Hi, is there a tutorial for using custom grids?

**Comment by dudaster** [comment]
No. But all you need to know is design the grid as you want and where do you want the post item to be just put post item widget placeholder.

Save it and then use it.

**Comment by jaykaslo** [comment]
Ohh… just realized… you’re replying to my comment on someone else’s issue (about working in google ads).  I just tried out creating a 3-column layout for the “custom grid” and yeah that’s cool.  I see how we can insert anything else we want in there (i.e. a spot for google ads, etc.)

I posted a separate issue though… about how Lottie animations are not displaying after infinite scrolling kicks in.  Please let me know re: this issue.  https://github.com/dudaster/ele-custom-skin/issues/271#issue-654266492

Thanks,
Jay Kaslo
